app/helpers/external_links_helper.rb

```diff
@@ -40,6 +40,22 @@ def edit_label_for(publishing_app:)
     end
   end
 
+  def feedex_url(from:, to:, base_path:)
+    host = Plek.new.external_url_for('support')
+    path = CGI.escape(base_path)
+    "#{host}/anonymous_feedback?from=#{from}&to=#{to}&paths=#{path}"
+  end
+
+  def google_analytics_url(from:, to:, base_path:)
+    from.delete!('-')
+    to.delete!('-')
+    base_path.gsub!(%r((\/)(?!\z)), '~2F')
```

Comment:
I do worry that using the `!` method variants could lead to subtle bugs... what do you think? Would it be worth avoiding the mutation?